osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/PreviewTimePart.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Game.Graphics;
+using osu.Game.Screens.Edit.Components.Timelines.Summary.Visualisations;
+
+namespace osu.Game.Screens.Edit.Components.Timelines.Summary.Parts
+{
+    public partial class PreviewTimePart : TimelinePart
+    {
+        protected override void LoadBeatmap(EditorBeatmap beatmap)
+        {
+            base.LoadBeatmap(beatmap);
+            Add(new PreviewTimeVisualisation(beatmap.PreviewTime));
+        }
+
+        private partial class PreviewTimeVisualisation : PointVisualisation
+        {
+            public PreviewTimeVisualisation(BindableInt time)
+                : base(time.Value)
+            {
+                time.BindValueChanged(s => X = s.NewValue);
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours) => Colour = colours.Lime;
```

Comment:
lime is already being used by difficulty points

https://github.com/ppy/osu/blob/60c8ef3fe5fe9adcb204814d975b7ed6c832fc8c/osu.Game/Beatmaps/ControlPoints/DifficultyControlPoint.cs#L31

Reply:
I'm not sure which color to use, I need advice

Reply:
Stable uses yellow
![image](https://user-images.githubusercontent.com/20495991/210075519-4c10086c-50dd-4b2f-9434-8c4f69da0de6.png)

Reply:
not sure yellow is going to work either as lazer uses that for timing points